frontend/src/components/Page4AllPaths.tsx

```diff
@@ -0,0 +1,158 @@
+import { useState } from "react";
+
+import { Button } from "./Button.tsx";
+import styles from "./Page4AllPaths.module.css";
+
+export function Page4AllPaths() {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const handleCheckboxChange = () => {
+    setIsChecked(!isChecked);
+  };
+
+  return (
+    <>
+      <div className={styles.page4Form}>
+        <form>
+          <h2 className={styles.sectionTitle}>Additional Information</h2>
+
+          <div className={styles.formRow}>
+            <label className={styles.label}>
+              Certified/Registered/Licensed in any other Jurisdiction
+              <input
+                className={styles.input}
+                type="text"
+                name="license"
+                placeholder="Certified/Registered/Licensed in any other Jurisdiction"
+              />
+            </label>
+
+            <label className={styles.label}>
+              Certified/Registered/License Number
+              <input
+                className={styles.input}
+                type="text"
+                name="licenseNumber"
+                placeholder="Certified/Registered/License Number"
+              />
+            </label>
+          </div>
+
+          <div className={styles.formRow}>
+            <label className={styles.label}>
+              Name of State/Jurisdiction
+              <input
+                className={styles.input}
+                type="text"
+                name="state"
+                placeholder="Name of State/Jurisdictionn"
+              />
+            </label>
+
+            <label className={styles.label}>
+              Date Certified/Registered/License Expires
+              <input
+                className={styles.input}
+                type="text"
+                name="license"
+                placeholder="Date Certified/Registered/License Expires"
+              />
+            </label>
+          </div>
+
+          <div className={styles.formRow}>
+            <label className={styles.label}>
+              Certification Exam
+              <input
+                className={styles.input}
+                type="text"
+                name="certificationExam"
+                placeholder="Certification Exam"
+              />
+            </label>
+
+            <label className={styles.label}>
+              Have you ever been convicted of a Felony?
+              <Button onClick={undefined} additionalStyle={styles.dropDown}>
```

Comment:
this should be a select element for dropdown functionality but i can also change it when i combine it with my form section component